libkbfs/disk_block_cache_test.go

```diff
@@ -717,6 +717,9 @@ func TestDiskBlockCacheUnsyncTlf(t *testing.T) {
 	seedDiskBlockCacheForTest(t, ctx, cache, config, numTlfs, numBlocksPerTlf)
 	require.Equal(t, numBlocks, standardCache.numBlocks)
 
+	standardCache.clearTickerDuration = 0
+	standardCache.numBlocksToEvictOnClear = 1
```

Comment:
Will setting these race with the loop?

Reply:
No, because this happens before the clearing loop starts (it's kicked off by the `SetTlfSyncState` below.

Reply:
Cool thanks.